images_3d.Rmd

```diff
@@ -108,20 +108,26 @@ data.shape
 Think of this as 256 2D images, stacked on top of one another.  Each 2D image
 is a "slice", of shape (256, 156).
 
-The first (bottom) slice:
+To get a sense of how a 2D slice looks like, let's visualize the middle slice:
 
 ```{python}
-bottom_slice = data[:, :, 0]
-plt.imshow(bottom_slice)
+middle_slice = data[:, :, img.shape[-1] // 2 - 1]
+plt.imshow(middle_slice)
 ```
 
-The middle slice:
+We can also *cut* the image through a different plane, for instance, sagittal:
```

Comment:
Here I'd delay until they have had more time to think about what the slicing means.

Reply:
As you found below, I thought bringing up `orthoview()` would be helpful for them to make their mind around 3D. Imaginging objects in 3D is hard, but it is easier if it is a head because we are used to look at them from many angles.

I don't think this can be confusing. Also, the sagittal plane is the easiest to interpret, which also helps make the argument here.